be/src/note/dto/create-note.input.ts

```diff
@@ -4,6 +4,6 @@ import { Column } from 'typeorm';
 @InputType()
 export class CreateNoteInput {
     @Field()
-    @Column({ length: 255, nullable: false })
+    @Column({ nullable: false })
```

Comment:
Oh wait, I did a lazy review, this shouldn't be here. the `@Column` decorator come from `typeorm` package and has nothing to do with gql input object. `@Column` is used in typeorm entities do describe db structure. This class here describes the input needed to gql to create a new Note.

Reply:
Fixed -> 6066234